docs/docker-container.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: base
+title: Docker Container Documentation
+---
+
+### Overview
+Clocker provides a base Docker container entity that will deploy a Docker Engine on any cloud / VM location. and run a Docker image automatically into a Docker container.
```

Comment:
Clocker provides a base Docker container entity that will deploy a Docker Engine and then start a selected Docker image automatically in it's Docker container. This entity can be deployed in any cloud / VM location.